root/static/scripts/relationship-editor/common/entity.js

```diff
@@ -145,13 +147,14 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
 
                                 if (currentValue && !newValue) {
                                     _.each(group.values.slice(0), function (r) {
-                                        r.linkOrder(0);
+                                         r.linkOrder(0) 
```

Comment:
You're indenting this one extra space, which seems to make it 5, and removing the semicolon for some reason. Just revert this (and line 154 below) to 4 spaces and a semicolon, please :)